README.md

```diff
@@ -83,3 +83,4 @@ make check
 
 - [ArthurHoaro](https://github.com/ArthurHoaro)
 - [tleon](https://github.com/tleon)
+- [GijsGoudzwaard](https://github.com/GijsGoudzwaard)
```

Comment:
Nice. Could you just reorder your name by alphabetic order ?

Reply:
Sure